docs/src/references/child-devices.md

```diff
@@ -0,0 +1,45 @@
+# Child Device Concept (Solution Proposal)
+
+Thin-edge provides a child-device concept to 
+TODO: Add motivation for child-device concept. Somethin like:
+-- Device-side and Cloud-side structural view (that might match the physical structure)
+-- Features are available per child-device (e.g. Software Management with one SW list per child device, ...)
+
+* Child-Device support is provided on thin-edge public interfaces as below:
```

Comment:
Before talking about the interfaces, we probably need a dedicated section on the data model that we support for child devices, explaining the kind of interactions that we expect between a child device and a connected cloud and the data model type that we propose for each interaction. And then we can do the mapping of which data model object maps to which interface.